index.js

```diff
@@ -290,6 +349,17 @@ const ContainerService = {
     ss.storage.identitiesData = this._identitiesState;
   },
 
+  sendTelemetryPayload(params = {}) {
+    // when pings come from popup, delete "method" prop
+    delete params.method;
+    let payload = { // eslint-disable-line prefer-const
+      "uuid": this._metricsUUID
+    };
+    Object.assign(payload, params);
+
+    this._sendEvent(payload);
```

Comment:
I wonder if we can have the same payload (or something super super similar) in each telemetry event.
Or move the complexity of creating the payload in 1 single place.

For instance, my initial idea was to have a method like this:

_sendEvent(eventType, userContextId, configuration) {
  let payload = Telemetry.createPayload(eventType, userContextId, configuration);
  ...
}

But we can do this later, if the complexity increases.